lib/streamlit/DeltaGenerator.py

```diff
@@ -2151,6 +2162,349 @@ def table(self, element, data=None):
 
         data_frame_proto.marshall_data_frame(data, element.table)
 
+    def show(self, *args):
```

Comment:
Use `@_wraps_with_cleaned_sig(method, 1)` to remove the `self` from this signature when printing to the user. It's a small thing but makes things a tiny bit clearer to users (since they have no idea `st.foo` actually creates a `DeltaGenerator` object, and that this `self` refers to that object).

That's the reason our API documentation doesn't have a bunch of `self` arguments. See here: https://streamlit.io/docs/api.html

Same below.

Reply:
Done